native/android/build.gradle

```diff
@@ -13,7 +13,8 @@ buildscript {
         // Config for MapLibre
         rnmbglMapboxLibs = {
             implementation("org.maplibre.gl:android-sdk:9.2.1")
-            implementation("com.mapbox.mapboxsdk:mapbox-sdk-turf:5.3.0")
+            implementation ("com.mapbox.mapboxsdk:mapbox-sdk-turf:5.8.0")
+            implementation 'com.mapbox.mapboxsdk:mapbox-android-telemetry:6.1.0'
```

Comment:
Do we really need this?

Reply:
thx for the hint. you're right. just removed it